docs/docs/api/nativeMethods/measure.md

```diff
@@ -35,3 +35,7 @@ const Comp = () => {
   return <View ref={aref} />;
 };
 ```
+
+### Note
+
+You can use `measure()` only on rendered components. Well practise is wrap function call with `try{} catch{}` if there exists possibility to call function on non rendered item, for example: Non visible item on screen from FlatList.
```

Comment:
Just a couple of remarks:
**Good** practi**c**e is **to** wrap
if there **is a** possibility
function on non rendered item => function on item which is not rendered
Non visible => invisible